mscore/preferences.h

```diff
@@ -76,7 +76,7 @@ enum class MusicxmlExportBreaks : char {
 
 // Default-zoom-type options
 enum class ZoomType : int {
-      PAGE_WIDTH = 0, WHOLE_PAGE, TWO_PAGES, PERCENTAGE
+      PERCENTAGE = 0, PAGE_WIDTH, WHOLE_PAGE, TWO_PAGES,
```

Comment:
is that reshuffling here and in prefsdialog.cpp really neccessary?

Reply:
This change keeps the default item at the top of the list, and also improves consistency with the order of the items in the toolbar zoom box.

Reply:
Technically yes, but in practice, not really. Since this preference is new for 3.5.0, it affects only people who were using 3.5.0 Alpha or Beta and who had also changed the default zoom type. I think that's a small price to pay for making everything internally consistent.

Reply:
Why does 0 need to be the default?
To me this PR looks too convoluted for a simple change of a single pref's default setting.
Seems a simple single line change from `static constexpr ZoomIndex startZoomIndex = ZoomIndex::ZOOM_PAGE_WIDTH;` to `static constexpr ZoomIndex startZoomIndex = ZoomIndex::ZOOM_100;` should be enough. Or amy I missung something?

Reply:
There are two reasons that zero is the default: First, it makes the combobox in the *MuseScore Preferences* dialog consistent with the toolbar zoom box (which lists percentages before “Page Width”, “Whole Page”, and “Two Pages”. Second, having the default item come first in the UI makes it look a little nicer, and also makes it consistent with the way the previous default item (“Page Width”) came first.

As to why this PR isn't a simple one-liner: I really wanted it to be, and I tried that first, but the one-liner didn't work. So I had to fix some bugs caused by some unfortunate assumptions scattered in the code that the default zoom type would always be “Page Width”.

With the bug fixes included in this PR, any future change to the default zoom type (if we should ever want to change it again) *will* be a one-liner, and it will be in `mscore/preferences.cpp` where it should be, and not hard-coded in various places in the code as it was before this PR.

Reply:
Should I go ahead and consider this resolved?

Reply:
I guess so, yes